app/policies/application_policy.rb

```diff
@@ -0,0 +1,49 @@
+class ApplicationPolicy
+  attr_reader :user, :record
+
+  def initialize(user, record)
```

Comment:
do you need all of these policies, stay in scope

Reply:
resolved